docs/book/src/sway-program-types/libraries.md

```diff
@@ -7,17 +7,17 @@ Libraries in Sway are files used to define new common behavior. The most promine
 Libraries are defined using the `library` keyword at the beginning of a file, followed by a name so that they can be imported.
 
 ```sway
-library my_library;
+library;
```

Comment:
I feel like we discussed this before, but can we just drop the keyword here altogether? At least for internal libraries? cc @FuelLabs/sway-compiler

Reply:
This is also relevant: https://github.com/FuelLabs/sway/issues/2428